core/src/main/java/com/microsoft/applicationinsights/internal/statsbeat/Feature.java

```diff
@@ -33,17 +33,19 @@
     JAVA_VENDOR_MICROSOFT(2),
     JAVA_VENDOR_ADOPT_OPENJDK(3),
     JAVA_VENDOR_REDHAT(4),
-    JAVA_VENDOR_OTHER(5);
+    JAVA_VENDOR_OTHER(5),
+    AAD_ON(6);
```

Comment:
let's call this the `AAD` feature, and it's either on `1` or off `0`
```suggestion
    AAD(6);
```